client/web/src/repo/commit/DiffModeSelector.tsx

```diff
@@ -1,9 +1,11 @@
+import classNames from 'classnames'
 import React from 'react'
 
 import { DiffMode } from './RepositoryCommitPage'
 
 interface DiffModeSelectorProps {
     className?: string
+    size?: 'small'
```

Comment:
nit: As this can only ever be a single string, I think it works better as a boolean

```suggestion
    small?: boolean
```

and then you can just do
`small && 'btn-sm'`